generators/hub/templates/package.json

```diff
@@ -1,20 +1,21 @@
 {
   "scripts": {
-    "build": "tsc -p .",
+    "build": "npm run lint && npm run clean && mkdir dist && webpack --progress --colors --config webpack.config.js --output-path ./dist -p",
```

Comment:
Are you required to run "mkdir" .. I'd expect webpack to create it if needed. Also, "mkdir" likely doesn't work on other platforms (e.g. Linux).